ci/travis_script_python.sh

```diff
@@ -34,11 +34,16 @@ source activate $CONDA_ENV_DIR
 python --version
 which python
 
-# faster builds, please
-conda install -y -q nomkl
-
-# Expensive dependencies install from Continuum package repo
-conda install -y -q pip numpy pandas cython
+conda install -y -q pip \
```

Comment:
https://github.com/pandas-dev/pandas/pull/17619

you may need to be more specific